lib/ProductOpener/Ecoscore.pm

```diff
@@ -102,7 +102,7 @@ sub load_agribalyse_data() {
 		$csv->getline ($io);
 
 		while ($row_ref = $csv->getline ($io)) {
-			$agribalyse{$row_ref->[0]} = {
+			$agribalyse{$row_ref->[0] . ''} = {
```

Comment:
Is this to make sure that we get a string? Couldn't we create a new "utils" method for this so it is more explicit? This way we could use it when we need without adding lots of comment explaining why do we do this.

Reply:
In fact Perl does it automatically for hash keys, I'll remove it.

Reply:
Removed